websites/A/Anghami/presence.ts

```diff
@@ -2,43 +2,66 @@ const presence = new Presence({
 		clientId: "721740741570986016",
 	}),
 	strings = presence.getStrings({
-		play: "general.playing",
+		play: "general.play",
 		pause: "general.paused",
-		browsing: "general.browsing",
+		browse: "general.browsing",
 	});
 
-const enum Assets {
-	Logo = "https://cdn.rcd.gg/PreMiD/websites/A/Anghami/assets/logo.png",
+function updateLogo() {
+	const trackCoverartDiv = document.querySelector(".track-coverart");
+	if (trackCoverartDiv) {
+		const style = trackCoverartDiv.getAttribute("style");
+		if (style) {
+			const match = decodeURIComponent(style).match(/url\("([^"]+?)"\)/);
+			if (match && match[1]) {
+				let url = match[1];
+				url = url.replace(/&size=\d+/, "");
+				return url;
+			}
+		}
+	} else {
```

Comment:
There is no else for the `if (style) {` and the `if (match && match[1]) {`, maybe instead of the else it should just be default if it hasn't returned yet set the logo asset?